### ISSUE TITLE
Detect device rotation even Orientation.lock is set?

### ISSUE THREAD

**Issue by KevKo1990** [yamill/react-native-orientation#234]
Hi,
I have the use case, that I want to rotate a picture in a modal floating over my normal view, however not the component beneath the modal.

Is there a way to detect rotations of the device even I set before `Orientation.lockToPortrait();`?
Thanks!

**Comment by sahil6111** [comment]
@KevKo1990 : were you able to achieve the above mentioned use case?
I want to resolve that in Android and it's working in iOS.

**Comment by khrizt** [comment]
@stoneman1 the @ronit-kadwane commit solves this problem, I can do a PR with that commit if you think it's ok

**Comment by DylanVann** [comment]
This would be useful to have. Sometimes you don't want to use the built in orientation events.

**Comment by johnyoonh** [comment]
@khrizt do you plan on doing the PR?

**Comment by khrizt** [comment]
@johnyoonh yep, here it is [PR#280](https://github.com/yamill/react-native-orientation/pull/280)

**Comment by MarcoPortillo** [comment]
Hello, I have the same use case, how can I use this PR?

**Comment by khrizt** [comment]
If you want you can use [our fork](https://github.com/coverfy/react-native-orientation) that includes theses changes.

**Comment by MarcoPortillo** [comment]
@khrizt yes, already done, works like a charm, thank you very much :)

**Comment by omar20261** [comment]
@khrizt  it helped me a lot , thanks man

**Comment by tolik85** [comment]
Implemented @khrizt sensor based orientation to iOS as well.
You can use my [my fork](https://github.com/tolik85/react-native-orientation).
